superset_patchup/version.py

```diff
@@ -0,0 +1,3 @@
+"""Version goes here - to avoid cyclic dependencies :-("""
```

Comment:
As soon as we start importing .views in __init__, pylint complains unless we do this.  Probably marginally cleaner anyway?